packages/webapp/public/locales/en/translation.json

```
@@ -1,6 +1,7 @@
 {
   "ADD_ANIMAL": {
     "ADD_ANIMALS": "Add animals",
+    "ADD_ANIMALS_FLOW": "add animals",
```

Grammatically in English this isn't quite correct ("Cancel your add animals") but I'm even more worried about the translators -- if they don't have the whole "cancel your {{flow}}" context (although this would be a great opportunity to add a screenshot!! I'll have to remember when uploaded to CrowdIn!) they will translate it as a verb action (same as the floating menu button) and it may not make any sense grammatically at all.
```suggestion
    "ADD_ANIMALS_FLOW": "addition of animals",
```

I don't love it, but I think it has to be a strict verbal noun here: "Cancel your addition of animals".

It's out of scope for now, but I wonder if we should revisit the modal alongside the _FLOW strings once the translations are done to make sure they work. I think I'll write a ticket because I'm also not that happy with the "your" in the modal -- "Cancel adding animals" would have been a lot cleaner in English and I think maybe in other languages too, but all the _FLOW strings would have to be checked against such a change 🫤 (And sorry that was a long comment about a translation!)

---

Thank you for catching, Joyce! I had checked other cancel flow titles, but I'm not sure how I ended up with this...😅 

We haven't created a release for translations!